backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/tokens/BpkColors.kt

```diff
@@ -64,10 +64,6 @@ public class BpkColors private constructor(
   public val surfaceDefault: Color,
   public val surfaceElevated: Color,
   public val surfaceHighlight: Color,
-  @Deprecated(DEPRECATION_MESSAGE)
```

Comment:
these are actually old, static colours that were treated as dynamic due to the detection for what's a dynamic token was wrong

Reply:
I think we need to clean up the foundations and remove all these custom tricks eventually

Reply:
eventually. but right now we still have usages of these (in the view system)